crux-lucene/test/crux/lucene/extension_test.clj

```diff
@@ -0,0 +1,26 @@
+(ns crux.lucene.extension-test
+  (:require  [clojure.test :as t]
```

Comment:
ah, I left more requires than strictly needed here, but I will make use of them in the next PR